convene-web/app/controllers/spaces_controller.rb

```diff
@@ -2,4 +2,8 @@ class SpacesController < ApplicationController
   def show
     @space = current_space
   end
+
+  def edit
+    @space = current_space
+  end
```

Comment:
Not blocking: Since we're using `space` in both `show` and `edit` we could expose a helper method instead that lazily retrieves the space, removing that responsibility from the show and edit actions:

```
helper_method def space
  @space ||= current_space
end
```

And replace the references to `@space` in the view  with `space`. 

As an aside,My personal preference is to leverage [decent_exposure](https://github.com/hashrocket/decent_exposure) for a consistent pattern for retrieval and building of objects for controllers.